components/EmailLogin.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useState } from 'react'
+
+export default function EmailLogin({onEmailSubmit, disabled}) {
+    const [email,setEmail] = useState('');
+
+
+    const handleSubmit = async(e: { preventDefault: () => void; }) => {
+        e.preventDefault();
+        onEmailSubmit(email);
+    }
+
+    return (
+        <>
+        <form onSubmit={handleSubmit} className="border rounded p-3 login-form">
```

Comment:
I generally think it's better to have styles defined in a separate file that is specific to this component. Then, here, you can use the class name so it's cleaner.

Reply:
Moved styles into a seperate file